Develop

leoboehm opened the pull request:
Added raspi file

eliasgroll commented on raspi.py:
```diff
@@ -0,0 +1,36 @@
+            except Exception as e:
+                print(f"Failed to send data: {e}")
+        else:
+            print("No Motion")
```
I think we do not want to print every second

leoboehm replied:
oh.. makes sense

eliasgroll commented on raspi.py:
```diff
@@ -0,0 +1,36 @@
+import RPi.GPIO as GPIO
```
Please write a test

eliasgroll commented on raspi.py:
```diff
@@ -0,0 +1,36 @@
+import RPi.GPIO as GPIO
```
please to the project add a service file so you can install that on the raspi as a service in case it crashes / needs to restart

eliasgroll replied:
https://medium.com/@benmorel/creating-a-linux-service-with-systemd-611b5c8b91d6

leoboehm commented:
**Changes:**
- added test file
- adapted flask server to receive motion state as true/false instead of setting timestamp
- removed prints

@eliasgroll please review again :)
